tests/storage/localfssd_test.py

```diff
@@ -879,6 +881,9 @@ def test_dump_sd_volumes(monkeypatch, tmp_repo, user_mount, user_domain):
         "version": str(user_domain.getVersion())
     }
 
+    expected_sequence = (42
+                         if user_domain.getVersion() >= 5
+                         else sc.DEFAULT_SEQUENCE)
```

Comment:
This works but is less readable than:

    if if user_domain.getVersion() >= 5:
        expected_sequence = 42
    else:
        expected_sequence = sc.DEFAULT_SEQUENCE

We use conditional expressions only if they fit in one line.

Reply:
ack